dolphinscheduler-task-plugin/dolphinscheduler-task-http/src/main/java/org/apache/dolphinscheduler/plugin/task/http/HttpTask.java

```diff
@@ -289,8 +293,20 @@ protected void setHeaders(HttpUriRequest request, List<HttpProperty> httpPropert
      */
     protected CloseableHttpClient createHttpClient() {
         final RequestConfig requestConfig = requestConfig();
-        HttpClientBuilder httpClientBuilder;
-        httpClientBuilder = HttpClients.custom().setDefaultRequestConfig(requestConfig);
+        HttpClientBuilder httpClientBuilder = HttpClients.custom();
+        if (httpParameters.getEnableProxy()) {
+            CredentialsProvider provider = new BasicCredentialsProvider();
+            AuthScope authScope = null;
+            // basic auth could tak effect alone without proxy
```

Comment:
Typo error here -> `tak`

Reply:
Address in [f1ce0d8](https://github.com/apache/dolphinscheduler/pull/14491/commits/f1ce0d8e1024727a3cb36b97932c4a2a596ad682) .